app/templates/components/forms/register-form.hbs

```diff
@@ -33,8 +33,12 @@
         </button>
       </div>
     </div>
-    <div class="ui secondary segment center aligned">
-      <a href="{{href-to 'login'}}" class="text muted weight-800">{{t 'Already have an account? Login'}}</a>
-    </div>
+    <a href="{{href-to 'login'}}">
```

Comment:
This is incorrect semantic markup. `segment` should be a direct child of `segments`